test/apps/test_deeplabv3p_ar_detect_app.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [[ $# < 2 ]]
+then
+    echo "usage: test_deeplabv3p_ar_detect_app.sh [app prefix] " \
+         "[data root] [mpi exec] [test cores]"
+    exit -1
+fi
+
+app_prefix=${1}
+data_root=${2}
+
+if [[ $# -eq 4 ]]
+then
+    mpi_exec=${3}
+    test_cores=${4}
+    launcher="${mpi_exec} -n ${test_cores}"
+    steps_per_file="--steps_per_file 4"
```

Comment:
Do we need a different number of steps per file in the MPI case? Unless there's a good reason to do this, it would be better to run the tests in the same way.

Reply:
Oh yea I did it before updating to Parallel-NetCDF. Will remove it